fedimint-server/src/consensus/transaction.rs

```diff
@@ -19,6 +20,21 @@ pub async fn process_transaction_with_dbtx(
         CONSENSUS_TX_PROCESSED_OUTPUTS.observe(out_count as f64);
     });
 
+    // We can not return the error here as errors are not returned in a specified
```

Comment:
@joschisan I have a crate that solves the problem of ordering. Also https://github.com/rayon-rs/rayon/issues/210 is worth looking at.

https://crates.io/crates/pariter